roles/ceph-rgw/tasks/multisite/create_realm_system_user.yml

```diff
@@ -0,0 +1,29 @@
+- name: check if the realm system user already exists
+  command: "{{ container_exec_cmd }} radosgw-admin user info --rgw-realm={{ rgw_realm }} --rgw-zonegroup={{ rgw_zonegroup }} --rgw-zone={{ rgw_zone }} --uid={{ rgw_realm_system_user }}"
+  delegate_to: "{{ groups[mon_group_name][0] }}"
+  register: usercheck
+  failed_when: False
+  changed_when: False
+  check_mode: no
+
+- name: create list realm_system_users
+  set_fact:
+    realm_system_users: "{{ realm_system_users | default([]) + [{ 'realm': hostvars[item]['rgw_realm'], 'zonegroup': hostvars[item]['rgw_zonegroup'], 'zone': hostvars[item]['rgw_zone'], 'access_key': hostvars[item]['system_access_key'], 'secret_key': hostvars[item]['system_secret_key'], 'user': hostvars[item]['rgw_realm_system_user'], 'display_name': hostvars[item]['rgw_realm_system_user_display_name'] }] }}"
+  with_items: "{{ groups.get('rgws', []) }}"
+  run_once: true
+  when: 
+    - hostvars[item]['rgw_zonemaster'] == true
```

Comment:
Don't use string comparison with boolean.
Use `| bool` instead (or `not ... | bool`)

Reply:
so it would be `hostvars[item]['rgw_zonemaster`]|bool == true`?

Reply:
no just `hostvars[item]['rgw_zonemaster'] | bool`